packages/js/LICENSE

```diff
@@ -0,0 +1,20 @@
+ESLint
+Copyright JS Foundation and other contributors, https://js.foundation
```

Comment:
```suggestion
Copyright OpenJS Foundation and other contributors, <www.openjsf.org>
```

To be the same as https://github.com/eslint/eslint/blob/main/LICENSE which was updated in https://github.com/eslint/eslint/pull/14877

(I'm guessing this was a copy & paste from eslint-config-eslint)